x-pack/plugin/ilm/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/ilm/CCRIndexLifecycleIT.java

```diff
@@ -650,20 +653,6 @@ private void putShrinkOnlyPolicy(RestClient client, String policyName) throws IO
                         builder.endObject();
                     }
                     builder.endObject();
-
-                    // Sometimes throw in an extraneous unfollow just to check it doesn't break anything
-                    if (randomBoolean()) {
```

Comment:
Why was this removed?

Reply:
Ah, I think I was suspecting this could cause some flakiness, but I believe it might've been due to dev issues as I was working through this feature (saving the execution state - namely the shrink index name being present in the execution state when we reach the cold pohase). I'll add this back. Thanks!